css-view-transitions-2/Overview.bs

```diff
@@ -755,7 +755,20 @@ div.box {
 
 # Nested view-transitions # {#nested-view-transitions}
 
-## Overview ## {#nested-overview}
+## Overview ## {#auto-name-overview}
+
+## Additions to 'view-transition-name' ## {#additions-to-vt-name}
+
+In addition to the existing values, the 'view-transition-name' also accepts an <dfn for=view-transition-name type=value export>auto</dfn> keyword.
+To resolve the [=used value=] of 'view-transition-name' for |element|:
+	1. Let |computed| be the [=computed value=] of 'view-transition-name'.
+	1. If |computed| is <css>none</css>, return null.
+	1. If |computed| is a <<custom-ident>>, return |computed|.
+	1. Assert: |computed| is <css>auto</css>.
+	1. If |element| has an associated [=Element/id=], then return |element|'s [=Element/id=].
+	1. Return a unique string, which is not a <<custom-ident>>. This string should remain consistent at least for the lifetime of |element|'s [=node document=]'s [=active view transition=].
```

Comment:
I'm not sure how exactly to reword this but a couple of things are not obvious right now:

- The string needs to uniquely identify the element. Right now it's ambiguous whether we mean unique for that specific transition.
- For the cross-doc case, this sounds like any `auto` element without an id will become an exit animation since there won't be a match in the new Document. Is that the intention? If so, the string should remain unique even the document changes.

Reply:
Done